R/sim_gs_n.R

```diff
@@ -329,6 +329,13 @@ sim_gs_n <- function(
 
     ans_1sim
   }
+
+  test_method <- ans$method[ans$sim_id == 1]
+  if (all(substr(test_method, 1, 3) == "WLR")) {
+    class(ans) <- c("simtrial_gs_wlr", class(ans))
+    attr(ans, "method") <- unique(ans$parameter[ans$sim_id == 1])
+  }
```

Comment:
It's my understanding that this class of `"simtrial_gs_wlr"` will be applied only when the test at each cutting is always `"WLR"`. Before we start adding classes and methods to {simtrial}, I think it would be good to discuss the plan for all the various classes and methods we envision adding. That way we can think through all the possibilities and then name them accordingly

Reply:
It's my understanding that this class of "simtrial_gs_wlr" will be applied only when the test at each cutting is always "WLR". [YZ: yes, you are correct.]

I think it would be good to discuss the plan for all the various classes and methods we envision adding. [YZ: to my point of view, there are 2 major tests, one is WLR and the other is MaxCombo. If we need further classification of WLR, then we could classify by weighting function, such FH, MB, ect..]

That way we can think through all the possibilities and then name them accordingly [Sure, let's discuss further next week.]

Reply:
I don't believe we ever had time to discuss this at a group meeting. Let's proceed, and we can always adjust the class names later